spirv_msl.cpp

```diff
@@ -14987,6 +15091,10 @@ bool CompilerMSL::OpCodePreprocessor::handle(Op opcode, const uint32_t *args, ui
 		break;
 	}
 
+	case OpTypeRayQueryKHR:
```

Comment:
Don't think we'll see types here. The ray tracing flag should be set based on SPIR-V capabilities I think, not based on opcodes.

Reply:
The Intersection Query supports procedural primitives by default, and there are no such procedural tags.